surah/src/main/res/drawable/ic_number_circle.xml

```diff
@@ -0,0 +1,27 @@
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <item>
+    <shape android:shape="oval">
+      <gradient
+          android:centerColor="#ffe2f1f6"
+          android:endColor="#ffe2f1f6"
+          android:gradientRadius="56dp"
+          android:startColor="#802BB5A8"
+          android:type="radial" />
+      <padding
+          android:bottom="6dp"
```

Comment:
Less padding looks the circle like clipped, 36dp is more pretty

Reply:
I just looked and didn't see any shadow in Figma design sample, so I'm going to remove it but if you insist I will use it